product_docs/docs/pgd/5/raft/03_migrating_to_raft_subgroups.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Migrating to Raft subgroups
+---
+
+This section assumes that you have an already running cluster and wish to introduce Raft subgroups to your cluster.
+
+
+## Migrating to Raft subgroups (without TPA)
+
+A number of configuration steps need to be taken to enable Raft subgroups to an existing cluster.
+
+* A “top-level” group needs to be defined for all nodes in the PGD cluster. Use bdr.create_node_group
```

Comment:
An *existing* cluster must have a top-level node group. So this first step is a bit at odds vs the "existing" part. Maybe just leave it away when considering an existing cluster?